core/templates/dev/head/components/ck-editor-helpers/ck-editor-4-rte.directive.ts

```diff
@@ -237,6 +239,8 @@ angular.module('oppia').directive('ckEditor4Rte', [
               break;
             }
           }
+          var data = elt.html();
+          $rootScope.$broadcast('changed', data);
```

Comment:
This is too invasive -- it affects the entire application. I think you'll need to find a cleaner approach that retrieves the data from CKEditor somehow, and that does not involve doing a rootScope broadcast.

Reply:
Sure, I'll look into this.